src/components/sidebar/SidebarNavigation/index.tsx

```diff
@@ -98,7 +101,7 @@ const Navigation = (): ReactElement => {
               <SidebarListItemText data-testid="sidebar-list-item" bold>
                 {item.label}
 
-                <SidebarListItemCounter count={getCounter(item)} />
+                <NavItemTag item={item} />
```

Comment:
You could pass queue size here to avoid calling the useQueuedTxsLength for each item.

Reply:
ok, i've refactored it.